core/config/ocr2_config.go

```diff
@@ -56,6 +52,7 @@ func (c *generalConfig) OCR2MonitoringEndpoint() string {
 func (c *generalConfig) OCR2KeyBundleID() (string, error) {
 	kbStr := c.viper.GetString(EnvVarName("OCR2KeyBundleID"))
 	if kbStr != "" {
+		// TODO: ID may not necesarily be a sha256 hash for solana
```

Comment:
I would say for now, ignore env vars entirely and focus exclusively on job spec configuration